CHANGELOG.md

```diff
@@ -6,9 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
 
-## [0.37.0] - 2023-09-19
```

Comment:
I believe you removed this line by mistake, can you move it to after the new line you've added below? It should be before line 13.

Reply:
I don't know why, but the previous build didn't work and blocked me. Do you know how to solve it?
https://github.com/vtex-apps/b2b-organizations-graphql/actions/runs/6241791489
https://github.com/vtex-apps/b2b-organizations-graphql/actions/runs/6248666170
https://github.com/vtex-apps/b2b-organizations-graphql/actions/runs/6248730389
https://github.com/vtex-apps/b2b-organizations-graphql/actions/runs/6248791700
https://github.com/vtex-apps/b2b-organizations-graphql/actions/runs/6248760613

Reply:
I did a rebase, and now it is worked.